index.html

```diff
@@ -2629,9 +2685,84 @@ <h2>Production and Consumption</h2>
 and consumed, including in JSON and JSON-LD.">
         <figcaption>
 Production and consumption of representations.
+See also: <a class="longdesc-link"
+href="#production-consumption-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+      <div class="longdesc" id="production-consumption-longdesc">
+        <p>
+The upper left quadrant of the diagram contains a rectangle with dashed grey
+outline, containing two blue-outlined rectangles, one above the other, as
+follows. The upper, larger rectangle is labeled, in blue, "Core Properties",
```

Comment:
```suggestion
outline, containing two blue-outlined rectangles, one above the other.
The upper, larger rectangle is labeled, in blue, "Core Properties",
```

Reply:
Applied in 63b89a2cf5b98cca535e84e39cc9cb68f7eeb738